libs/island-ui/core/src/lib/IconRC/icons/UploadOutline.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react'
+import type { SvgProps as SVGRProps } from '../types'
+
+const SvgUploadOutline = ({
+  title,
+  titleId,
+  ...props
+}: React.SVGProps<SVGSVGElement> & SVGRProps) => {
+  return (
+    <svg
+      className="download-outline_svg__ionicon"
```

Comment:
className is download, this should prob be upload right?

Reply:
omg yeah ofc